### PR TITLE
ENG-3824 amend params attribute for bundle descriptor

### PR DESCRIPTION
**Pull request by jeffgo10**

### CODE REVIEW

File: src/models/bundle-descriptor-constraints.ts [hunk 1]
```diff
@@ -155,6 +156,17 @@ export const NAV_CONSTRAINTS: ObjectConstraints<Nav> = {
   }
 }
 
+export const PARAM_CONSTRAINTS: ObjectConstraints<WidgetParams> = {
+  name: {
+    required: true,
+    type: 'string'
+  },
+  description: {
+    required: true,
```
Comment (cecchisandrone) [inline src/models/bundle-descriptor-constraints.ts]: not required

File: src/models/bundle-descriptor.ts [hunk 2]
```diff
@@ -32,6 +32,11 @@ export enum SecurityLevel {
   Lenient = 'lenient'
 }
 
+export type WidgetParams = {
```
Comment (cecchisandrone) [inline src/models/bundle-descriptor.ts]: WidgetParam

File: src/models/bundle-descriptor-constraints.ts (hunk 1 above)
Comment (jeffgo10) [reply src/models/bundle-descriptor-constraints.ts]: done

File: src/models/bundle-descriptor.ts (hunk 2 above)
Comment (jeffgo10) [reply src/models/bundle-descriptor.ts]: done

**Review by cecchisandrone — APPROVED** [review]

**Pull request merged**